src/index.js

```diff
@@ -5,17 +5,23 @@ import * as serviceWorker from "./serviceWorker";
 import "semantic-ui-css/semantic.min.css";
 import axios from "axios";
 import { BrowserRouter } from "react-router-dom";
-import { StripeProvider } from 'react-stripe-elements'
+import { StripeProvider } from "react-stripe-elements";
 import "./css/index.css";
+import { Provider } from "react-redux";
+import configureStore from "./state/store/configureStore";
 
 axios.defaults.baseURL = "http://localhost:3000/api";
+const store = configureStore();
+window.store = store;
 
 ReactDOM.render(
-  <StripeProvider apiKey="pk_test_21nBNjeqdyB1Mzm2VjDPQprF00kyEKYZSK">
-    <BrowserRouter>
-      <App />
-    </BrowserRouter>
-  </StripeProvider>,
+  <Provider store={store}>
+    <StripeProvider apiKey="pk_test_21nBNjeqdyB1Mzm2VjDPQprF00kyEKYZSK">
```

Comment:
are you not using an env file for the API key?

Reply:
That's the publishable key, do we need to put it in a env file?

Reply:
This is merged from upstream...

Reply:
will put in in env file